AndelaOSP/Andela-Socials
# #166153666 Add MVP features to develop

#### What Does This PR Do?
- This PR introduces all the implementation and fixes built for the MVP into the staging branch to be deployed to the staging environment
#### Description Of Task To Be Completed
- [x] Make all MVP features and fixes available in the staging environment
#### Any Background Context You Want To Provide?
- N/A
#### How can this be manually tested?
- Start the application
- Test to see that all features in the `mvp-develop` branch have been successfully introduced into the application
#### What are the relevant pivotal tracker stories?
[#166153666](https://www.pivotaltracker.com/story/show/166153666)
#### Screenshot(s)
